src/pages/ProfileCloseAcountPage.jsx

```diff
@@ -0,0 +1,45 @@
+import ButtonProfile from "../shared/ButtonProfile";
+import ProfileHeader from "../shared/ProfileHeader";
+
+const ProfileCloseAcountPage = () => {
+  const borderStyle = "border border-gray-400";
+  const title = "Close Account";
+  const description = "Close your account permanently.";
+  return (
+    <div>
+      <section className={`py-8 border-b-0 ${borderStyle}`}>
+        <ProfileHeader title={title} description={description} />
+        <div className="px-3 pb-12 border-b border-gray-400 md:px-5 lg:px-10">
+          <div className="flex flex-wrap justify-between gap-5 py-5">
+            <p className="text-l md:text-xl">
+              Suspending account activity for an indefinite period
+            </p>
+
+            <select
+              name=""
```

Comment:
It's good to add name and id attributes here for select element or even for input fields, that will be useful when connect with a server.